.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "javascript.format.enable": true,
```

Comment:
This is to ensure that our code is formatted using prettier if we're using VSCode.